crates/forge-runner/src/build_trace_data/test_program_path.rs

```diff
@@ -0,0 +1,46 @@
+use anyhow::Context;
+use anyhow::Result;
+use cairo_lang_sierra::program::VersionedProgram;
+use camino::Utf8PathBuf;
+use fs4::FileExt;
+use std::fs;
+use std::fs::File;
+use std::io::BufWriter;
+
+pub const TESTS_PROGRAMS_DIR: &str = ".snfoundry_test_code";
+
+#[derive(Clone)]
+pub struct TestProgramPath(Utf8PathBuf);
+
+impl TestProgramPath {
```

Comment:
Aren't those contracts as well? Also would add `Sierra` prefix to make it clear.

Would be nice to provide a comment what it exactly is.

Reply:
No, they are not contracts, I don't see how they would be

Reply:
I think we generally distinct those in ohter parts of the codebase, would be nice to do it here as well.

Reply:
I don't understand what you mean here, it is distinct from contracts